airgap/uds-dev-stack/zarf.yaml

```diff
@@ -0,0 +1,85 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-dev-stack-airgap-images
+  yolo: true
+  description: >
+    *** REQUIRES DOCKER ***
+    Loads the required images for UDS Dev Stack into Docker to be used offline.
+
+constants:
+  - name: LOCAL_PATH_PROVISIONER_VERSION
+    value: "v0.0.31"
+  - name: BUSYBOX_VERSION
+    value: "latest"
+  - name: WOLFI_BASE_VERSION
+    value: "latest"
+  - name: PAUSE_VERSION
+    value: "3.10"
+  - name: NGINX_VERSION
+    value: "1.28.0-alpine3.21"
+  - name: METALLB_CONTROLLER_VERSION
+    value: "v0.14.9"
+  - name: METALLB_SPEAKER_VERSION
+    value: "v0.14.9"
+  - name: FRR_VERSION
+    value: "9.1.0"
+  - name: MINIO_VERSION
+    value: "RELEASE.2025-04-03T14-56-28Z"
+  - name: MINIO_MC_VERSION
+    value: "RELEASE.2024-11-21T17-21-54Z"
```

Comment:
General comment - all of these should be captured by renovate to automate maintenance of this package (this would also apply to the other zarf.yaml file as well).